Source/Programs/CLI.h

```diff
@@ -12,7 +12,8 @@ inline std::vector<uint8> loadFile(const char* filename)
 	std::ifstream stream(filename,std::ios::binary | std::ios::ate);
 	if(!stream.is_open())
 	{
-		std::cerr << "Failed to open " << filename << std::endl;
+		std::string errorStr = std::string("Failed to open file '") + filename + "'";
+		perror(errorStr.c_str());
```

Comment:
Could you make this do something like this instead?
```
std::cerr << "Failed to open file '" << filename < "': " << std::strerror(errno) << std::endl;
```

I didn't know `perror` without googling it, so I think using `std::strerror` is a bit easier to read.